src/SeleniumLibrary/__init__.pyi

```diff
@@ -133,7 +133,7 @@ class SeleniumLibrary:
     def select_all_from_list(self, locator: Union[selenium.webdriver.remote.webelement.WebElement, str]): ...
     def select_checkbox(self, locator: Union[selenium.webdriver.remote.webelement.WebElement, str]): ...
     def select_frame(self, locator: Union[selenium.webdriver.remote.webelement.WebElement, str]): ...
-    def select_from_list_by_index(self, locator: Union[selenium.webdriver.remote.webelement.WebElement, str], *indexes: str): ...
+    def select_from_list_by_index(self, locator: Union[selenium.webdriver.remote.webelement.WebElement, str], *indexes: int): ...
```

Comment:
Does this need to be `Union[str, int]` for Robot to do the type conversion?

Reply:
And if robot core does not do the type conversion I would be fine with both 

Union[str, int] as the argument type and then inrternally in all spots needed, something like

```
if isinstance(indexes, str):
    int(indexes)
```

.. although now I see these can be and are *indexes and not index, meaning it could be multiple indexes .. thus complicating my conversion strategy a bit ..

Reply:
[Noting these comments apply to all instances of these str to ints]

Reply:
Corner cases are not know but rf will do type conversion to int

Union is used only for with alias_or_index because alias is user defined string  bu index is int returned by create_driver and register_driver